File: TP1 - C++/data/models/payment_consultation.cpp
#include "../../domain/entities/payment_consultation.hpp"

PaymentConsultation::PaymentConsultation(string id, double value, string patientName, string paymantDate):
    id(id),
    value(value),
    patientName(patientName),
    paymentDate(paymentDate){};
PaymentConsultation::~PaymentConsultation(){}

string PaymentConsultation::getId(){
    return this->id;
}
double PaymentConsultation::getValue(){
    return this->value;
}
void PaymentConsultation::setValue(double value){
    this->value = value;
}

File: TP2 - Java/src/domain/entities/PaymentConsultation.java
package domain.entities;

public class PaymentConsultation {
    private String id;
    private String patientName;
    private String paymentDate;
    private double value;

    public PaymentConsultation(String id, String patientName, String paymentDate, double value) {
        this.id = id;
        this.patientName = patientName;
        this.paymentDate = paymentDate;
        this.value = value;
    }

    public String getId() {
        return id;
    }

    public String getPatientName() {
        return patientName;
    }

    public void setPatientName(String patientName) {
        this.patientName = patientName;
    }

    public String getPaymentDate() {
        return paymentDate;
    }

    public void setPaymentDate(String paymentDate) {
        this.paymentDate = paymentDate;
    }

    public double getValue() {
        return value;
    }

    public void setValue(double value) {
        this.value = value;
    }
}
